notify/notify.go

```diff
@@ -40,6 +40,18 @@ func (GrowlNotifier) Push(title string, text string, iconPath string) {
 	go cmd.Run()
 }
 
+// OSXNotifier is a notifier for osx's Notification Center
+type OSXNotifier struct {
+}
+
+// Push implements Notifier
+func (OSXNotifier) Push(title string, text string, _ string) {
+	cmd := exec.Command(
+		`osascript -e "display notification '`, text, `' with title '`, title, `'"`,
```

Comment:
The argument decomposition here is wild. Isn't something like the following needed to make it work at all?

	cmd := exec.Command(
		"osascript",
		"-e",
		fmt.Sprintf(`display notification "%v" with title "%v"`, text, title))